### PR TITLE
added a bundle for TMLR

### PR DESCRIPTION
**Pull request by philipphennig**
I have added a bundle for [TMLR](https://jmlr.org/tmlr/), based on the [official template](https://www.overleaf.com/latex/templates/tmlr-journal-submissions/mwchznnhdtwx). It's largely based on the JMLR template, with slightly wider text (6.5 inches, instead of 6), and fontsize 10 instead of 10.95.

### CODE REVIEW

File: tueplots/figsizes.py [hunk 1]
```diff
@@ -182,6 +182,36 @@ def jmlr2001(
...
+    nrows=1,
+    ncols=2,
+    constrained_layout=True,
+    tight_layout=False,
+    height_to_width_ratio=_GOLDEN_RATIO,
+    pad_inches=_PAD_INCHES,
+):
+    """TMLR figure size.
+
+    Source: https://www.overleaf.com/latex/templates/tmlr-journal-submissions/mwchznnhdtwx
```
Comment (pnkraemer) [inline tueplots/figsizes.py:197]: Should we link

https://www.jmlr.org/tmlr/author-guide.html

instead of overleaf? .

**Review by pnkraemer — COMMENTED** [review]
Thanks!

Would be great if you could add a test case to tests/test_rc_params_cases/case_bundles.py before we merge. Other than that, we're good to go.

File: tueplots/figsizes.py (hunk 1 above)
Comment (philipphennig) [reply tueplots/figsizes.py:197]: yeah, you're right.

**Review by pnkraemer — APPROVED** [review]
Thanks!

**Comment by LucasAlegre** [comment]
Hi! I just tried the TMLR bundle, and the font size are way smaller than the font sizes in the latex template of TMLR. Did it change in 2024 by any chance?

**Comment by pnkraemer** [comment]
Hi @LucasAlegre, thanks for raising this! :) Could you create a new issue? Thanks!

**Pull request merged**